.github/workflows/cicd.yml

```diff
@@ -0,0 +1,248 @@
+# Docs:
+# https://help.github.com/en/actions/automating-your-workflow-with-github-actions
+
+
+
+name: CI/CD
+
+on: [push]
```

Comment:
Should also run on pull requests. And on cron is a decent idea too IMO, which was never possible to configure in the travis.yml but thankfully in GHA is doable inline here.

E.g.: https://github.com/Julian/jsonschema/blob/master/.github/workflows/ci.yml

Reply:
What value does cron add?

Reply:
Helps discover build flakiness earlier, especially if e.g. a dep goes unpinned and a bulid doesn't run for awhile -- the cron will run anyhow and you'll get an email the morning after your unpinned thing broke your CI.

Basically increases the total number of runs, and if you do it at a time when no one commits (middle of the night say) then it's "for free" and doesn't block other runs that may want to execute.

Reply:
(Don't let this count as blocking feedback of course by the way in case that needed to be said:) was simply a suggestion, if you disagree just ignore)

Reply:
I'm going to punt on cron, since we're doing fine without it now, and it's an easy thing to add later.

Reply:
Does it make sense to only run on `pull_request`?  By enabling both, when you have a PR, everything runs twice, which seems rather wasteful.

Reply:
Removing `push` for now…